test/testenv/verificationutils.go

```diff
@@ -1162,3 +1162,23 @@ func VerifyIsDeploymentInProgressFlagIsSet(ctx context.Context, deployment *Depl
 		return isDeploymentInProgress
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
 }
+
+// VerifyFilesInFolder verify that files are present in folder. Set checkAppDirectory false to verify file is not present.
+func VerifyFilesInFolder(ctx context.Context, deployment *Deployment, testenvInstance *TestCaseEnv, ns string, podNames []string, files []string, path string, checkAppDirectory bool) {
+	for _, podName := range podNames {
+		gomega.Eventually(func() bool {
+			// Using checkAppDirectory here to get all files in case of negative check.  GetDirsOrFilesInPath  will return files/directory when checkAppDirecotry is FALSE
+			filelist, err := GetDirsOrFilesInPath(ctx, deployment, podName, path, checkAppDirectory)
+			gomega.Expect(err).To(gomega.Succeed(), "Unable to get files on pod", "Pod", podName)
+			for _, file := range files {
+				folderName := file + "/"
+				found := CheckStringInSlice(filelist, folderName)
+				logf.Log.Info("App check", "On pod", podName, "check app", folderName, "is in path", path, "Status", found)
```

Comment:
Also, this comment is not correct, probably a copy-paste error.

Reply:
Modified